src/plugins/intel_gpu/src/kernel_selector/cl_kernels/fully_connected_gpu_bf_tiled.cl

```diff
@@ -1117,7 +1124,7 @@ KERNEL(fc)(
 #endif
 ) {
 #if USE_SLM
-    #if DYNAMIC_QUANTIZE
+    #if DYNAMIC_QUANTIZE && (TILE_OFM == 2)
```

Comment:
this condition looks weird. dynamic quantization supports `tile_ofm==2` only? If so, `DYNAMIC_QUANTIZE == 0` when `TILE_OFM != 2`. Then we don't need to change this line..

Reply:
tile_ofm is selected by dispatch data from autotuneparam. Then should_dynamic_quantize should check dispatch data. I thought this would be more simple than checking dispatch data.

Reply:
Let's sync offline

Reply:
from jitter: MAY_DYNAMIC_QUANTIZE

#if MAY_DYNAMIC_QUANTIZE && TILE_OFM == 2
#define DYNAMIC_QUANTIZE 1
#else
#define DYNAMIC_QUANTIZE 0
#endif